src/map/Marker.tsx

```diff
@@ -28,7 +28,7 @@ export function MarkerComponent({ color, number, size = 0 }: MarkerProps) {
             xmlns="http://www.w3.org/2000/svg"
             viewBox="0 0 384 512"
             height={size}
-            style={{ ...markerStyle, fill: color, transform: `translate(${-size / 2}px, ${-size}px` }}
```

Comment:
This was broken in multiple ways: First of all, the closing parenthesis of the `translate` function was missing so this `transform` did not do anything at all. Second, if anything we only want to translate in the vertical direction (not horizontally). Third, we want to translate vertically only by half of the size of the symbol (the origin is at the center without translation, but the tip of the marker is at the bottom). Finally, size is `0` here currently, so the translation won't do nothing even after fixing the aforementioned issues. Why is size 0? And why do we still see the markers (I mean in the sidebar. For the markers in the map we do not use `Marker.tsx` currently (see #179))?

Reply:
It could be that we set the width of the element from the svg's container. Though the element's style should be applied last 🤔

Reply:
Ok, but for now / as long as we do not use `Marker.tsx` for the map markers I think we can just remove this transform/translate because apparently it did not do anything.